tasks/parse-pdr/index.js

```diff
@@ -89,7 +89,7 @@ exports.handler = handler;
 
 // use node index.js local to invoke this
 justLocalRun(() => {
-  const payload = require('@cumulus/test-data/cumulus_messages/parse-pdr.json'); // eslint-disable-line global-require, max-len
+  const payload = require('@cumulus/test-data/cumulus_messages/parse-pdr.json'); // eslint-disable-line global-require
```

Comment:
Same issue if this payload is ever modified in any way. Can recreate the reference location convenience with something like `fs.readFileSync(require.resolve('@cumulus/...'))`